lib/fortnox/api/models/base.rb

```diff
@@ -19,8 +19,12 @@ def self.attribute( name, *args )
         end
 
         def self.new( hash )
-          obj = preserve_meta_properties( hash ) do
-            super( hash )
+          begin
+            obj = preserve_meta_properties( hash ) do
+              super( hash )
+            end
+          rescue Dry::Struct::Error, Dry::Types::ConstraintError, Dry::Types::MissingKeyError => e
```

Comment:
We moved a lot of code to the `Types` module, and `Types::Model` raises `Fortnox::API::MissingAttributeError`, so I just wanted to make sure that this logic should be placed in `Model::Base`.